corehq/apps/registration/forms.py

```diff
@@ -79,8 +79,12 @@ class RegisterWebUserForm(forms.Form):
     is_mobile = forms.BooleanField(required=False, widget=forms.HiddenInput())
 
     def __init__(self, *args, **kwargs):
+        self.is_sso = kwargs.pop('is_sso', False)
         super(RegisterWebUserForm, self).__init__(*args, **kwargs)
 
+        if self.is_sso:
```

Comment:
Would it make sense to check `ENFORCE_SSO_LOGIN` here to prevent an accidental bypass of password requirement if `is_sso` somehow was true here when `ENFORCE_SSO_LOGIN` is not set? I know that shouldn't be possible, but it seems more consistent with the check you're doing in `clean_password`.

```suggestion
        if settings.ENFORCE_SSO_LOGIN and self.is_sso:
```